### PR TITLE
[4.0] Update composer/ca-bundle to remove expiret - DST Root CA X3

### PR DESCRIPTION
**Pull request by wojsmol**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/33518#issuecomment-937977203.

### Summary of Changes
Similar to https://github.com/joomla/joomla-cms/pull/35781 in 4.0-dev and up we are using composer/ca-bundle directly

Remove old expired DST Root CA X3 Let's Encrypt root certificate - see [Old Let’s Encrypt Root Certificate Expiration and OpenSSL 1.0.2](https://www.openssl.org/blog/blog/2021/09/13/LetsEncryptRootCertExpire/)

### Testing Instructions
Make HTTPS request to server using latest Let's Encrypt certificate from server using openssl 1.0.2

### Actual result BEFORE applying this Pull Request
HTTPS connections to servers with latests Let's Encrypt certificates from server with openssl version 1.0.2 are NOT working.

### Expected result AFTER applying this Pull Request
HTTPS connections to servers with latests Let's Encrypt certificates from server with openssl version 1.0.2 are working.

### Documentation Changes Required
N/A

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -9739,5 +9739,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.1.0"
+    "plugin-api-version": "2.0.0"
```
Comment (richard67) [inline composer.lock:9742]: Where does this downgrade come from? Maybe an unrelated change? If so, it should be reverted.

Comment (wojsmol) [reply composer.lock:9742]: This is auto generated by `Composer version 2.0.13 2021-04-27 13:11:08` - maybe I have older composer version then last person updating dependencies.

**Comment by wilsonge** [comment]
Closing this in favour of the full file update in https://github.com/joomla/joomla-cms/pull/35785/files

**Comment by wilsonge** [comment]
Woops this was the 4.0 PR - my bad - reopening

File: composer.lock (hunk 1 above)
Comment (wilsonge) [reply composer.lock:9742]: Probably - worth running a `composer self-update` probably - but doesn't block this PR

**Comment by wilsonge** [comment]
Merging on review. Thanks!
